packages/widgets/src/components/line/line-chart.widget.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { number, text, object } from '@storybook/addon-knobs';
+import {
+  cardKnobs,
+  axisXKnobs,
+  axisYKnobs,
+  gridKnobs,
+  typographyKnobs,
+  legendKnobs,
+  curveKnobs,
+  groupModeKnobs,
+  lineStackModeKnobs,
+} from '@keen.io/storybook-utils';
+import { Typography } from '@keen.io/ui-core';
+import { theme as keenTheme } from '@keen.io/charts';
+
+import { LineChartWidget } from './line-chart.widget';
+import { chartData } from './line-chart.widget.fixtures';
+import { createLabelFormatter } from './line-chart.widget.utils';
```

Comment:
I'll put it to story as this is only internal thing to demonstrate formatting possibilities.